drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodListDialogFragment.kt

```diff
@@ -73,7 +73,7 @@ class PaymentMethodListDialogFragment : DropInBottomSheetDialogFragment(), Payme
                         ImageLoader.getInstance(requireContext(), mDropInViewModel.dropInConfiguration.environment),
                         arguments?.getBoolean(SHOW_IN_EXPAND_STATUS)!!,
                         this)
-                recyclerView.layoutManager = LinearLayoutManager(requireContext())
+                recyclerView.layoutManager = androidx.recyclerview.widget.LinearLayoutManager(requireContext())
```

Comment:
nit: imports?  (same in a few other source files)

Reply:
Thx for pointing this out, I will fix this on another PR.